Dockerfile.rhel7

```diff
@@ -0,0 +1,11 @@
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
+WORKDIR /go/src/github.com/openshift/cluster-image-registry-operator
+COPY . .
+RUN make build
+
+FROM registry.svc.ci.openshift.org/ocp/4.0:base
```

Comment:
any particular reason you didn't go with `openshift/ocp-4.0:base` to stay aligned w/ the origin image naming pattern?

Reply:
Because the repo has to be private.  Origin instead will be moved.